lib/Resque/Job/Status.php

```diff
@@ -39,7 +39,14 @@ class Resque_Job_Status
 	 */
 	public function __construct($id)
 	{
-		$this->id = $id;
+		$this->id = self::generateId($id);
+	}
+
+	/**
+	 * generate job id consistently
+	 */
+	private static function generateId($id) {
```

Comment:
Done and done. Though, I see a number of other classes in the project that have functions with inconsistent opening braces, so didn't realize there was a consistent style to follow.